### ISSUE TITLE
Deduplicate / simplify the structure

### ISSUE THREAD

**Issue by Sophist-UK** [rdswift/picard-docs#40]
**Type of change requested**
This is an…
 - [x] Error Report
 - [ ] Addition Request
 - [ ] Deletion Request
 - [ ] Modification Request
 - [ ] Other

**Reason for the change**
The Extending Picard section contains information about Plugins and scripts which are covered elsewhere in the Using Picard section. This is confusing IMO and should be consolidated.

Similarly the technicalities of Plugins are duplicated and are really for people who want to develop / contribute and are advanced topics. 

And Workflow Recommendations are good but seem to me to be a detailed part of Using Picard.

**Describe the solution you'd like**
A clear and concise description of what you want changed.

**Additional context**
Add any other context or screenshots about the change request here.

**Comment by rdswift** [comment]
I agree that it looks like a bunch of topics patched together without a lot of organizational consideration, because that's exactly what it is.  :smile:  Can you suggest an outline of the proposed structure that I can work towards?  Thanks.

**Comment by Sophist-UK** [comment]
Sorry - but no time to review the current structure in detail, but...

I think the overall structure needs to cover in increasing expertise:

1. Basic use without any scripts or options.

2. Options (without covering actual scripting but describing default file naming structure)

3. Finding, installing, updating , uninstalling Plugins + a summary of current official plugins.

4. Scripts
4a. File Naming scripts overview with examples
4b. Tagging scripts overview with examples
4c. Tags / Variables - general, detailed list, media type mapping
4d. Functions - general, detailed list with examples

5. Writing plugins
5a. Writing plugins to add script functions
5b. Writing other plugins inc. list of registrations and when they are called.
5c. Specifics for web calls to get more online data - i.e. how to make / queue a call and create a callback function, (& I can add some stuff for) caching and avoiding multiple simultaneous calls for the same data etc.

I have probably missed a lot, but I hope this gives you the gist.

**Comment by rdswift** [comment]
I've created a [wiki page](https://github.com/rdswift/picard-docs/wiki/User-Guide-Document-Restructuring-Proposal) that we should all be able to edit, with a proposed outline for the restructured document.  I'm hoping that we can achieve some sort of consensus on the structure of the end product before starting the actual work of rewriting it.
